README.md

```diff
@@ -1271,6 +1271,7 @@ See also [Are we game yet?](https://arewegameyet.rs)
 ### Graph algorithms
 
 * [s1ck/graph](https://github.com/s1ck/graph) - A library for high-performant graph algorithms [![graph CI status](https://img.shields.io/github/workflow/status/s1ck/graph/CI/main?label=CI)](https://github.com/s1ck/graph/actions/workflows/rust.yml)
+* [petgraph/petgraph](https://github.com/petgraph/petgraph) - Graph data structure library for Rust. [![graph CI status](https://github.com/petgraph/petgraph/workflows/Continuous%20integration/badge.svg?branch=master)](https://github.com/petgraph/petgraph/actions/workflows/ci.yml)
```

Comment:
Notice `p` is before `s` in the alphabet.

Besides that, it have more than 50 stars
![image](https://user-images.githubusercontent.com/36519478/220618428-b7e23683-298e-4784-9471-bc4c46cb7b63.png)